src/main/java/me/coley/cafedude/attribute/StackMapTableAttribute.java

```diff
@@ -0,0 +1,383 @@
+package me.coley.cafedude.attribute;
+
+/**
+ * <p>
+ * Used during the process of verification by type checking.
+ * </p>
+ * 
+ * <p>
+ * There may be at most one StackMapTable attribute in the attributes table of
+ * a Code attribute.
+ * </p>
+ * 
+ * <p>
+ * In a class file whose version number is 50.0 or above, if a method's Code
+ * attribute does not have a StackMapTable attribute, it has an implicit stack
+ * map attribute (§4.10.1). This implicit stack map attribute is equivalent to
+ * a StackMapTable attribute with number_of_entries equal to zero.
+ * </p>
+ * 
+ * @author x4e
+ */
+public class StackMapTableAttribute extends Attribute {
+	public final StackMapFrame[] frames;
```

Comment:
For simplicity of allowing easy insertions/removals I've been using `List<Type>`.